### PR TITLE
Minor: added sidebar

### PR DESCRIPTION
**Pull request by carriemacfarlane**
#### Summary

Added sidebar

#### Motivation

WTD Writing Day

#### Metadata

- [ ] Adds a new document
- [ ] Rewrites (or significantly expands) a document
- [x] Fixes a typo, bug, or other error

### CODE REVIEW

File: files/en-us/web/api/css/factory_functions/index.md [hunk 1]
```diff
@@ -10,6 +10,7 @@ tags:
   - factory function
 browser-compat: api.CSS
 ---
+{{APIRef(“CSSOM”)}}
```
Comment (wbamberg) [inline files/en-us/web/api/css/factory_functions/index.md:13]:
```suggestion
{{APIRef("CSSOM")}}
```

**Review by wbamberg — CHANGES_REQUESTED** [review]
Thanks for your PR, @carriemacfarlane !

We apparently need to use straight quotes for macro arguments. Sorry, this is my fault I think, because in the spreadsheet I used the other kind of quotes.

**Comment by carriemacfarlane** [comment]
@wbamberg Thanks!

**Review by wbamberg — COMMENTED** [review]
👍 thanks for the update, @carriemacfarlane !

**Review by wbamberg — APPROVED** [review]

**Pull request merged**